framework/include/cppmicroservices/BundleActivator.h

```diff
@@ -127,11 +127,15 @@ struct BundleActivator
  * \snippet uServices-activator/main.cpp 0
  */
 #define CPPMICROSERVICES_EXPORT_BUNDLE_ACTIVATOR(_activator_type)              \
+  extern "C" US_ABI_EXPORT cppmicroservices::BundleActivator*                  \
+    US_CREATE_ACTIVATOR_FUNC(US_BUNDLE_NAME)();                                \
```

Comment:
Add single line comment to explain why the declaration is included

Reply:
Deciding not to do this, the inclusion of that line should be obvious (regarding what it syntactically means (prototype definition)) (even if somebody doesn't know what warning we are trying to avoid)